_data/packages.yml

```diff
@@ -123,7 +162,7 @@
   reviewers:
     - name: Tom Aldcroft
       github_username: taldcroft
-    - name: Michele Delli Veneri
+    - name:
```

Comment:
Similar to above people's names are being randomly?? erased for some reason.

Reply:
https://github.com/MicheleDelliVeneri This user's name (as opposed to Chiara) isn't listed in the GitHub profile. BUT i thought we had added logic to handle that if the name was already in this file it should not overwrite it.